Linking broken with aslam_optimizer

@eggerk The new changes broke how the suitesparse package links stuff, which in turn causes aslam_optimizer packages to not compile because of missing headers/libraries.

**Environment:**
Ubuntu 14.04 with Indigo / Ubuntu 16.04 with Kinetic
Clean catkin workspace, with merged devel space and only extending /opt/ros/\<ROS-VERSION\>

**Steps to reproduce:**
catkin build sparse_block_matrix
(similar error also for the other packages in the aslam_optimizer)

**Issue:**
Build fails with error
`sparse_helper.h:27:21: fatal error: cholmod.h: No such file or directory`

Which is a header file in SuiteParse, the linking of which should be handled by this package. If manually added to CPATH, compilation will fail with missing references to functions during linking.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -43,7 +43,7 @@ install(DIRECTORY ${CATKIN_DEVEL_PREFIX}/lib/
                        PATTERN "libsuitesparseconfig*"
                        PATTERN "libumfpack*"
 )
-cs_export(INCLUDE_DIRS ${CATKIN_DEVEL_PREFIX}/include
+cs_export(INCLUDE_DIRS ${CATKIN_DEVEL_PREFIX}/include/suitesparse
           LIBRARIES amd
                     btf
                     camd

